libs/platform/src/lib/components/form/input/input.component.ts

```diff
@@ -86,4 +86,8 @@ export class InputComponent extends BaseInput implements OnInit, AfterViewInit {
             throw new Error(` Input type ${this.type} is not supported`);
         }
     }
+    /** @hidden */
+    ngAfterViewInit(): void {
+        this.inputElemRef.nativeElement.style.display = 'block';
+    }
```

Comment:
Can we set the style display in a `.scss` file? or is there a reason that we need to do this in the `ngAfterViewInit` hook?